cypress/integration/pages/articles/testsForAMPOnly.js

```diff
@@ -43,21 +43,80 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
     // `appToggles` tells us whether a feature is toggled on or off in the current environment.
     if (appToggles.mediaPlayer.enabled) {
       describe('Media Player', () => {
-        it('should render a placeholder image', () => {
-          cy.request(`${config[service].pageTypes.articles.path}.json`).then(
-            ({ body }) => {
-              // `video` blocks can also contain audio.
-              const media = getBlockData('video', body);
-              if (media) {
-                cy.get('div[class^="StyledVideoContainer"]').within(() => {
-                  cy.get('amp-img')
-                    .should('have.attr', 'src')
-                    .should('not.be.empty');
-                });
-              }
-            },
-          );
-        });
+        // This test is being temporarily throttled to the service 'news'.
+        if (service === 'news') {
+          it('should have a Media Player', () => {
```

Comment:
looking into this

Reply:
We decided to limit some of the playback tests for canonical pages (e.g. https://github.com/bbc/simorgh/blob/latest/cypress/integration/pages/articles/testsForCanonicalOnly.js#L143) because we were concerned about them being being flaky (well, I was). In practice they don't seem to have been especially flaky so we could consider running them for a few more services, but I think that should be a separate PR.